Lib/test/test_tokenize.py

```diff
@@ -1908,9 +1908,6 @@ def test_random_files(self):
 
         testfiles.remove(os.path.join(tempdir, "test_unicode_identifiers.py"))
 
-        # TODO: Remove this once we can untokenize PEP 701 syntax
-        testfiles.remove(os.path.join(tempdir, "test_fstring.py"))
-
         for f in ('buffer', 'builtin', 'fileio', 'os', 'platform', 'sys'):
             testfiles.remove(os.path.join(tempdir, "test_%s.py") % f)
```

Comment:
These also all seem to pass, though I'm not sure why they're currently removed (there's no comment indicating why)